src/timeAgoInWords.README.md

```diff
@@ -0,0 +1,37 @@
+
+# `timeAgoInWords`
+
+Takes an absolute date in [epoch format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description) and returns a string representing relative time ago.
+
+## Usage
+`timeAgoInWords(epoch, opts)`
+
+Returns: `string | false`
+
+Converts an absolute epoch to a relative time ago string
+
+`epoch`
+
+Type: `number`
+
+The date when an event happened in epoch format
+
+`opts`
+
+Type:
+```typescript
+{
+	length?: 'short' | 'med' | 'long'; // Change string length (default 'med')
```

Comment:
is this because you think `length` is clearer or to remove an instance of `format`?

length to me suggests how many characters i'll get back (`String#length`) rather than the format of the sentence?

Reply:
I changed `format` to `length` because I realised using 'format' here is wrong for the _same_ reasons it is wrong elsewhere and, more importantly, it will be hard to keep Typescript happy if we were to try to import this lib into DCR or AR where `format: Format` means something else.

But yeah, arguably, `length` also suffers from similar problems where it is a common property and appears a lot in code but it's not [a reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#keywords), communicates intent and reads nicely. It is true though that it has a connotation around numbers of charaters. `FormatLength`?

Reply:
ah yeah right of course! 

is it `WordsFormat`?

Reply:
it seems really weird that no two packages can use the same name for a type, surely TS must be able to handle that?